rbi/parlour.rbi

```diff
@@ -0,0 +1,879 @@
+      def create_includes(includables); end
+
+      sig { params(name: String, value: String, block: T.nilable(T.proc.params(x: Constant).void)).returns(Constant) }
+      def create_constant(name, value:, &block); end
+
+      sig { params(name: String, type: String, block: T.nilable(T.proc.params(x: Constant).void)).returns(Constant) }
+      def create_type_alias(name, type:, &block); end
+
+      sig { override.overridable.params(others: T::Array[RbiGenerator::RbiObject]).returns(T::Boolean) }
+      def mergeable?(others); end
+
+      sig { override.overridable.params(others: T::Array[RbiGenerator::RbiObject]).void }
+      def merge_into_self(others); end
+
+      sig { override.overridable.returns(String) }
+      def describe; end
+
+      sig { overridable.params(indent_level: Integer, options: Options).returns(T::Array[String]) }
+      def generate_body(indent_level, options); end
+
+      sig { params(object: RbiObject).void }
+      def move_next_comments(object); end
+    end
+
+    class Options
+      extend T::Sig
+
+      sig { params(break_params: Integer, tab_size: Integer, sort_namespaces: T::Boolean).void }
+      def initialize(break_params:, tab_size:, sort_namespaces:); end
+
+      sig { returns(Integer) }
+      attr_reader :break_params
+
+      sig { returns(Integer) }
+      attr_reader :tab_size
+
+      sig { returns(T::Boolean) }
+      attr_reader :sort_namespaces
+
+      sig { params(level: Integer, str: String).returns(String) }
+      def indented(level, str); end
+    end
+
+    class Parameter
+      extend T::Sig
+      PREFIXES = {
+        normal: '',
+        splat: '*',
+        double_splat: '**',
+        block: '&'
+      }.freeze
+
+      sig { params(name: String, type: T.nilable(String), default: T.nilable(String)).void }
+      def initialize(name, type: nil, default: nil); end
+
+      sig { params(other: Object).returns(T::Boolean) }
+      def ==(other); end
+
+      sig { returns(String) }
+      attr_reader :name
+
+      sig { returns(String) }
+      def name_without_kind; end
+
+      sig { returns(String) }
+      attr_reader :type
+
+      sig { returns(T.nilable(String)) }
+      attr_reader :default
+
+      sig { returns(Symbol) }
+      attr_reader :kind
+
+      sig { returns(String) }
+      def to_def_param; end
+
+      sig { returns(String) }
+      def to_sig_param; end
+    end
+
+    class RbiObject
+      abstract!
+
+      extend T::Helpers
+      extend T::Sig
+
+      sig { params(generator: RbiGenerator, name: String).void }
+      def initialize(generator, name); end
+
+      sig { returns(RbiGenerator) }
+      attr_reader :generator
+
+      sig { returns(T.nilable(Plugin)) }
+      attr_reader :generated_by
+
+      sig { returns(String) }
+      attr_reader :name
+
+      sig { returns(T::Array[String]) }
+      attr_reader :comments
+
+      sig { params(comment: T.any(String, T::Array[String])).void }
+      def add_comment(comment); end
```

Comment:
hey y'all, still wrapping my head around how bundled rbis worked, but I came across a small issue here while upgrading `sorbet-rails` to use the latest `parlour` (https://github.com/chanzuckerberg/sorbet-rails/pull/367). 

it looks like there is an alias method here https://github.com/AaronC81/parlour/blob/66e31b9ab26493dad149f833dc844b18806bebab/lib/parlour/rbi_generator/rbi_object.rb#L74 which isn't being included in the rbi. is this a bug in parlour's rbi generation or a deliberate omission?

Reply:
This looks like an oversight in the `TypeParser`; it doesn't seem to handle aliases, but that should definitely be added.